py/server/deephaven/partitioned_table.py

```diff
@@ -0,0 +1,1175 @@
+        """Applies the table 'sort_descending' operation to all constituent tables of the underlying partitioned 
+        table, and produces a new PartitionedTableProxy with the result tables as the constituents of its underlying 
+        partitioned table. 
+
+        Args:
+            order_by (Union[str, Sequence[str]]): the column(s) to be sorted on
+
+        Returns:
+            a new PartitionedTableProxy
+
+        Raises:
+            DHError
+        """
+        try:
+            order_by = to_sequence(order_by)
+            return PartitionedTableProxy(j_pt_proxy=self.j_pt_proxy.sortDescending(*order_by))
+        except Exception as e:
+            raise DHError(e, "sort_descending operation on the PartitionedTableProxy failed.") from e
+
+    def where(self, filters: Union[str, Filter, Sequence[str], Sequence[Filter]] = None) -> PartitionedTableProxy:
+        """Applies the table 'where' operation to all constituent tables of the underlying partitioned table, 
+        and produces a new PartitionedTableProxy with the result tables as the constituents of its underlying 
+        partitioned table. 
+
+        Args:
+            filters (Union[str, Filter, Sequence[str], Sequence[Filter]], optional): the filter condition
+                expression(s) or Filter object(s), default is None
+
+        Returns:
+            a new PartitionedTableProxy
+
+        Raises:
+            DHError
+        """
+        try:
+            filters = to_sequence(filters)
+            return PartitionedTableProxy(j_pt_proxy=self.j_pt_proxy.where(*filters))
+        except Exception as e:
+            raise DHError(e, "where operation on the PartitionedTableProxy failed.") from e
+
+    @auto_locking_op
+    def where_in(self, filter_table: Table, cols: Union[str, Sequence[str]]) -> PartitionedTableProxy:
+        """Applies the table 'where_in' operation to all constituent tables of the underlying partitioned table with the
+        provided filter table, and produces a new PartitionedTableProxy with the result tables as the constituents
+        of its underlying partitioned table.
+
+        Args:
+            filter_table (Table): the table containing the set of values to filter on
+            cols (Union[str, Sequence[str]]): the column name(s)
+
+        Returns:
+            a new PartitionedTableProxy
+
+        Raises:
+            DHError
+        """
+        try:
+            cols = to_sequence(cols)
+            return PartitionedTableProxy(j_pt_proxy=self.j_pt_proxy.whereIn(filter_table.j_table, *cols))
+        except Exception as e:
+            raise DHError(e, "where_in operation on the PartitionedTableProxy failed.") from e
+
+    @auto_locking_op
+    def where_not_in(self, filter_table: Table, cols: Union[str, Sequence[str]]) -> PartitionedTableProxy:
+        """Applies the table 'where_not_in' operation to all constituent tables of the underlying partitioned table
```

Comment:
Another comment on this repeated docs string.  Can we make the docs link to the table method?